### DISCUSSION TITLE
VALL-EX what do you think?

### DISCUSSION THREAD

**Discussion by ghost** [lifeiteng/valle#69] [Ideas]
the paper VALL-EX is released, that could be great idea to across the challenge in code-switch synthesis.
do you consider to apply it?

**Comment by lifeiteng** [comment]
@primepake Yes, I'll implement it in next month.

**Comment by ghost** [comment]
when you start the project?

**Reply by lifeiteng** [reply]
I'll start in next week, finish it in one or two weeks.

**Reply by ghost** [reply]
Amazing

**Reply by ghost** [reply]
How's the progress in Vallex? also NeuralSpeech2 is released the paper that looks like more real than valle
https://speechresearch.github.io/naturalspeech2/

**Reply by evanbrociner** [reply]
@lifeiteng Any progress on Valle-x by any chance? Thank you!!!!
